src/components/Survey/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+export const AnythingWrongSurvey = () => {
+  const { t } = useTranslation();
+  return (
+    <div className="margin-top-4">
+      <p>
+        <a
+          href="https://www.surveymonkey.com/r/DM6NYRX"
+          target="_blank"
+          rel="noopener noreferrer"
+          aria-label="Open EASi survey in a new tab"
+        >
+          {t('general:feedback.anythingWrong')}
+        </a>
+      </p>
+    </div>
+  );
+};
+
+/* There are multiple versions of the Improve EASi survey depending on format of
+  page it is being added to */
+
+// This is the common version for most pages
+export const ImproveEasiSurvey = () => {
+  const { t } = useTranslation();
+  // Period and space to be added between translations since they are on same line
+  const periodSpace = '. ';
+  return (
+    <div className="margin-top-4">
+      <p>
+        {t('general:feedback.improvement') + periodSpace}
+        <a
+          href="https://www.surveymonkey.com/r/JNYSMZP"
+          target="_blank"
+          rel="noopener noreferrer"
+          aria-label="Open EASi survey in a new tab"
+        >
+          {t('general:feedback.whatYouThink')}
+        </a>
+      </p>
+    </div>
+  );
+};
+
+/* This version is only for decision pages, it has a header so that it fits
+  with the theme of the pages */
+export const ImproveEasiDecisionPageSurvey = () => {
```

Comment:
This should not be its own component because:
1. It's not a "common" component that is reused throughout the application. The naming is specific to a certain view/page.
2. It's only used once. We don't need to make this abstraction this early if there's only one use case. We can put this markup directly into the view.

Reply:
I imagine that this will be on all "decision" pages, right now that is only two but we will most likely have more pages that need this format of survey. If we want to wait until further pages are added to do that, I can make that change. it also makes the other impove easi survey easier to style since we dont have to support this header style